src/main/java/core/basesyntax/service/FileServiceImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileServiceImpl implements FileService {
+    @Override
+    public List<String> readFile(String fileName) {
+        List<String> result;
+        try {
+            result = Files.readAllLines(Path.of(fileName));
```

Comment:
```suggestion
            return Files.readAllLines(Path.of(fileName));
```
you can return the result right away